package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
can we point to the new Headless-React version?

Reply:
Updated!